http-scala-fx/src/main/scala/fx/HttpRetries.scala

```diff
@@ -0,0 +1,36 @@
+package fx
+
+/**
+ * Models HttpRetries as an Int
+ */
+opaque type HttpRetries = Int
```

Comment:
I guess `HttpRetries` was thought to be used along with the `HttpRetryPolicy` class, but, checking out the code, I see that the `HttpRetryPolicy` function is defined as `(HttpResponse, Int) => Boolean,` so this type is not being used in the codebase. Should we remove it? Or change the function signature to use it rather than an `Int` value?

Reply:
Yes, that was an oversight. It should be HttpRetries.

Reply:
And as I‌ commented above, the maximum retries should also be configurable and passed to the policy function.

Reply:
Resolved in [ce2b2a4](https://github.com/47deg/TBD/pull/37/commits/ce2b2a4da2a01b983a46e3a5ab23f80770d74d24)